_includes/tutorials/multi-joins/ksql/code/tutorial-steps/dev/populate-table-items.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO items VALUES ('1', 'Television 60-in');
```

Comment:
It would be helpful to differentiate the values in `items` and `customers`, instead of both having sample data `1, 2, 3`....maybe make `items` be something else, like `101, 102, 103`?

Reply:
changed in latest